### PR TITLE
Fix packages test case

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1-fixed pep8 issues in multiple places
2- Test case as this test only applicable on powerpc
so added check if this test case run on other than powerpc
test cancel
Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
this PR address #728

File: ras/packages.py [hunk 1]
```diff
@@ -24,6 +24,10 @@
 class Package_check(Test):
 
     def setUp(self):
+
```
Comment (kamalesh-babulal) [inline ras/packages.py]: Minor nit, can the extra line be removed ?

**Comment by kamalesh-babulal** [comment]
@PraveenPenguin if you add `Fixes: #728` to the commit message above the signed-off-by line, that issue automatically gets closed once this commit is merged. I have not go through the complete code but the changes looks correct.

File: ras/packages.py (hunk 1 above)
Comment (PraveenPenguin) [reply ras/packages.py]: @kamalesh-babulal sure will do

**Comment by PraveenPenguin** [comment]
@narasimhan-v can you please have a look on this PR

File: ras/packages.py [hunk 2]
```diff
@@ -38,8 +38,9 @@ def test(self):
                 'packages_rhel', default=['lshw', 'librtas'])
             self.packages.extend(packages_rhel)
             for package in self.packages:
-                if "anaconda" in process.system_output("yum list installed | "
-                                                       "grep %s | tail -1" % package, shell=True):
+                if "anaconda" in process.system_output("yum list installed "
+                                                       "| grep %s | tail -1"
+                                                       % package, shell=True):
```
Comment (narasimhan-v) [inline ras/packages.py]: @PraveenPenguin Can this be turned into python please ?

**Comment by PraveenPenguin** [comment]
@narasimhan-v pushed the changes please have look

**Comment by PraveenPenguin** [comment]
@narasimhan-v ah sorry for that , push the changes

**Pull request merged**